source/structured_data.cc

```diff
@@ -604,6 +607,174 @@ namespace aspect
     }
 
 
+
+    template <int dim>
+    void
+    StructuredDataLookup<dim>::load_netcdf(const std::string &filename)
+    {
+#ifndef ASPECT_WITH_NETCDF
+      (void)filename;
+      AssertThrow(false, ExcMessage("Loading NetCDF files is only supported if ASPECT is configured with the NetCDF library!"));
+#else
+      TableIndices<dim> new_table_points;
+      std::vector<std::string> coordinate_column_names(dim);
+      std::vector<std::string> data_column_names;
+      std::vector<Table<dim,double>> data_tables;
+      std::vector<std::vector<double>> coordinate_values(dim);
+
+      int ncid;
+      int status;
+
+      status = nc_open(filename.c_str(), NC_NOWRITE, &ncid);
+      AssertThrow(status == NC_NOERR, ExcMessage("NetCDF Error occured!"));
```

Comment:
Might be useful at some point to add an AssertNetCDF like we have AssertMPI and friends.

Reply:
done.